src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm_copy_b.hpp

```diff
@@ -0,0 +1,148 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "emitters/plugin/x64/jit_emitter.hpp"
+#include "emitters/snippets/jit_snippets_call_args.hpp"
+#include "emitters/snippets/cpu_kernel_executor_table.hpp"
+
+#include <cpu/x64/brgemm/brgemm.hpp>
+#include <cpu/x64/matmul/brgemm_matmul_copy_utils.hpp>
+
+
+namespace ov {
+namespace intel_cpu {
+
+struct BrgemmCopyBKernelConfig : public snippets::KernelExecutorBase::GenericConfig {
+public:
+    BrgemmCopyBKernelConfig() = default;
+    BrgemmCopyBKernelConfig(const element::Type& src_dt, const element::Type& wei_dt, dnnl_format_tag_t format,
+                            dnnl::impl::cpu::x64::cpu_isa_t isa, bool is_with_comp, bool is_transposed_B, dnnl_dim_t wei_N_blk);
```

Comment:
Actually, `format` and `is_transposed_B` parameters are equivalent. Maybe we can store only one of them in the config?